src/executePrettier/buildPrettierOptions.js

```diff
@@ -10,11 +12,15 @@ const {
   isCurrentScopeVueScope,
 } = require('../editorInterface');
 const { shouldUseEditorConfig, getPrettierOptions, getAtomTabLength } = require('../atomInterface');
-const buildEditorConfigOptions = require('./buildEditorConfigOptions');
 const { getPrettierInstance } = require('../helpers');
 
 const isDefined: (x: any) => boolean = _.negate(_.isNil);
 
+const buildEditorConfigOptions: (file: FilePath) => Object = _.flow(
```

Comment:
prefer `{}` over `Object` in general:

The return type used to be this for our internal one, this is probably pretty close:

```js
-type TranslatedOpts = {	
-  tabWidth?: number,	
-  printWidth?: number,	
-  useTabs?: boolean,	
-};	
```

Reply:
> {} over Object

Ah didn't know about that.  I haven't used flow before so thanks.
<br>

> The return type

Hmm - I purposefully left it at Object (or `{}`) to leave it flexible for changes in the dependency.  Given the yarn.lock file I suppose bumps in version allow us to hardcode its return type.  I'll do this

Reply:
Which I much prefer to all the requests made by users to authors to support their favorite flavor of js typing haha